image/secure-boot/genkeys.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# Copyright (c) Edgeless Systems GmbH
+#
+# SPDX-License-Identifier: AGPL-3.0-only
+
+# This script generates a PKI for secure boot.
+# It is based on the example from https://github.com/systemd/systemd/blob/main/man/loader.conf.xml
+# This is meant to be used for development purposes only.
+# Release images are signed using a different set of keys.
+# Set PKI to an empty folder and PKI_SET to "dev".
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+TEMPLATES=${SCRIPT_DIR}/templates
+BASE_DIR=$(realpath "${SCRIPT_DIR}/..")
+if [ -z "${PKI}" ]; then
+    PKI=${BASE_DIR}/pki
+fi
+if [ -z "${PKI_SET}" ]; then
+    PKI_SET=dev
+fi
+
+gen_pki () {
+    # Only use for non-production images.
+    # Use real PKI for production images instead.
+    count=`ls -1 ${PKI}/*.{key,crt,cer,esl,auth} 2>/dev/null | wc -l`
```

Comment:
```suggestion
    count=$(ls -1 ${PKI}/*.{key,crt,cer,esl,auth} 2>/dev/null | wc -l)
```

Reply:
Shellcheck also suggests using `find` instead of `ls`, not sure if it fits here, though